### PR TITLE
Subaru: add FW for 2019 Subaru Impreza

### PR DESCRIPTION
**Pull request by seven121-ym**
I have an Impreza 2019 model in Japan.
It is almost the same as the already registered Impreza 2019.
However, since Ecu.esp and Ecu.eps are different, add them.

dongle id:7f6bb63c061b59cc

### CODE REVIEW

File: selfdrive/car/subaru/values.py [hunk 1]
```diff
@@ -88,6 +89,7 @@ class CAR:
       b'z\xc0\x04\x00',
       b'z\xc0\x00\x00',
       b'\x8a\xc0\x10\x00',
+      b'z\xc0\x08\x00',
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/values.py:92]:
```suggestion
```

Comment (seven121-ym) [reply selfdrive/car/subaru/values.py]: Why is it unnecessary to add a line for Ecu.eps?
"b'z \ xc0 \ x08 \ x00',"
It was present in my car log. Will Openpilot work without this?

Comment (adeebshihadeh) [reply selfdrive/car/subaru/values.py]: It was a duplicate. All of your car's FW versions are now in master.

Comment (seven121-ym) [reply selfdrive/car/subaru/values.py]: もし私が正しく理解できていなかったとしたら、ごめんなさい。
"b'z\xc0\x08\x00',"は"values.py"の中にまだ記載されていないと思う。
似ている"b'z\xc0\x04\x00',"はリストにありますが。

Comment (seven121-ym) [reply selfdrive/car/subaru/values.py]: If I didn't understand it correctly, I'm sorry.
I think "b'z \ xc0 \ x08 \ x00'," is not yet listed in "values.py".
A similar "b'z \ xc0 \ x04 \ x00'," is in the list.

Comment (adeebshihadeh) [reply selfdrive/car/subaru/values.py]: It's equivalent to `b'z\xc0\b\x00'`

Comment (seven121-ym) [reply selfdrive/car/subaru/values.py]: Thank you for answering many times.
That's because of the character code.

**Pull request merged**